src/components/SellerList.js

```diff
@@ -4,27 +4,18 @@ import styled from 'styled-components';
 import { API } from '../config';
 
 const SellerList = props => {
-  const [sellerList, setSellerList] = useState([]);
-
-  useEffect(() => {
-    fetch(`${API.SELLER}`, {
-      method: 'GET',
-    })
-      .then(response => response.json())
-      .then(result => {
-        setSellerList(result.seller_list);
-      });
-  }, []);
-
   const history = useHistory();
 
+  const [heartCheck, setHeart] = useState([]);
+  const heartArr = [];
+
   return (
     <AllWrap>
       <SellerTotalBox>
-        <SellerTotal>셀러 ({sellerList.length})</SellerTotal>
+        <SellerTotal>셀러 ({props.sellerList.length})</SellerTotal>
```

Comment:
- 위에서 구조분해할당 하시면 다음과 같이 작성될 것 같습니다.
```suggestion
        <SellerTotal>셀러 ({sellerList.length})</SellerTotal>
```

Reply:
넵넵!